src/components/Scheme/ContributionRewardExtRewarders/Competition/Details.tsx

```diff
@@ -0,0 +1,331 @@
+                <div className={css.bullet}></div>
+                <div className={css.label}>Competition start time:</div>
+                <div className={css.datetime}>{formatFriendlyDateForLocalTimezone(startTime)}</div>
+              </div>
+              <div className={css.period}>
+                <div className={css.bullet}></div>
+                <div className={css.label}>Submission end time:</div>
+                <div className={css.datetime}>{formatFriendlyDateForLocalTimezone(submissionsEndTime)}</div>
+              </div>
+              <div className={css.period}>
+                <div className={css.bullet}></div>
+                <div className={css.label}>Voting start time:</div>
+                <div className={css.datetime}>{formatFriendlyDateForLocalTimezone(votingStartTime)}</div>
+              </div>
+              <div className={css.period}>
+                <div className={css.bullet}></div>
+                <div className={css.label}>Competition end time:</div>
+                <div className={css.datetime}>{formatFriendlyDateForLocalTimezone(endTime)}</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        
+        <div className={css.submissions}>
+          <div className={css.heading}>{submissions.length}&nbsp;Submissions</div>
+          <div className={css.list}>
+            {submissionsHtml()}
+          </div>
+        </div>
+      </div>
+    
+      {this.state.showingCreateSubmission ?
+        <Modal onBackdropClick={this.cancelNewSubmissionModal}>
+          <CreateSubmission
+            proposalState={proposalState}
+            daoState={daoState}
+            handleCancel={this.cancelNewSubmissionModal}
+            handleSubmit={this.submitNewSubmissionModal}></CreateSubmission>
+        </Modal> : ""
+      }
+
+      {this.state.showingSubmissionDetails ?
+        <Modal onBackdropClick={this.closeSubmissionDetailsModal}
+          backdropClassName={css.submissionsModalBackdrop}>
+          <SubmissionDetails
+            currentAccountAddress={this.props.currentAccountAddress}
+            suggestionId={this.state.showingSubmissionDetails.id}
+            proposalState={proposalState}
+            daoState={daoState}
+            handleClose={this.closeSubmissionDetailsModal}
+            handleVote={this.voteOnSubmission}
+            handleRedeem={this.redeemSubmission}></SubmissionDetails>
+        </Modal> : ""
+      }
+
+    </React.Fragment>;
+  }
+}
+
+const CompetitionDetailsConnected = connect(mapStateToProps, mapDispatchToProps)(CompetitionDetails);
+
+export default withSubscription({
+  wrappedComponent: CompetitionDetailsConnected,
+  loadingComponent: null,
+  errorComponent: (props) => <div>{ props.error.message }</div>,
+  checkForUpdate: [],
+  createObservable: (props: IExternalProps & IStateProps ) => {
+    /**
+     * HEADS UP:  we subscribe here because we can't rely on getting to this component via CompetitionCard
```

Comment:
???
(I mean, what you are saying here seems ok, I'm jus tnot sure why we need a "HEADS UP" warning here...)